pkg/controller/config/config.go

```diff
@@ -59,6 +59,7 @@ type Config struct {
 	WorkflowReEval      config.Duration      `json:"workflow-reeval-duration" pflag:"\"30s\",Frequency of re-evaluating workflows"`
 	DownstreamEval      config.Duration      `json:"downstream-eval-duration" pflag:"\"60s\",Frequency of re-evaluating downstream tasks"`
 	LimitNamespace      string               `json:"limit-namespace" pflag:"\"all\",Namespaces to watch for this propeller"`
+	NamespaceFilter     string               `json:"namespace-filter" pflag:",Regular expression to apply fine grain filter when watching all namespaces"`
```

Comment:
```suggestion
	NamespaceFilterRegex     string               `json:"namespace-filter" pflag:",Regular expression to apply fine grain filter when watching all namespaces"`
```

Reply:
Will fix in a commit for code alignment sake.